lexicon/conversion.py

```diff
@@ -81,10 +81,38 @@ def run(self):
 
 
 class LexiconUniqueOrthJob(Job):
-    def __init__(self, bliss_lexicon):
+    """Merge lemmata with the same orthography.
+
+    Merging strategy (only natural proper word lemmata are affected)
+
+        1. which lemmata are merged
+            case 1:  merge_multi_orths_lemmata == False
+                only lemmata with exactly one orth element are affected
+            case 2:  merge_multi_orths_lemmata == True
+                lemmata with more than one orth will also be considered
+                for checking whether two (or more) lemmata share the same orth,
+                we only compare the PRIMARY orth, since otherwise the merging
+                logic can become ambiguous and cumbersome
+
+        2. how lemmata are merged
+            orth/phon/eval
+                all orth/phon/eval elements are merged together
+            synt
+                synt element is only copied to final lemma when
+                    1) the final lemma does not already have one
+                    2) and the rest to-be-merged-lemmata have any synt element
+                    ** having a synt <=> synt is not None
+                this could lead to INFORMATION LOSS if there are several
+                different synt token sequences in the to-be-merged lemmata
+    """
+
+    __sis_hash_exclude__ = {"merge_multi_orths_lemmata": False}
+
+    def __init__(self, bliss_lexicon, merge_multi_orths_lemmata=False):
```

Comment:
```suggestion
    def __init__(self, bliss_lexicon, merge_multi_orths_lemmata=False):
            """
        :param tk.Path bliss_lexicon: lexicon file to be handeled
        :param bool merge_multi_orths_lemmata: if True, also lemmata containing
            multiple orths are merged based on their primary orth. Otherwise they
            are ignored.
        """
```